book/src/getting-started/4-qml-gui.md

```diff
@@ -11,7 +11,7 @@ As noted in the [QObjects in Rust](./1-qobjects-in-rust.md) chapter, we always w
 For a small modern GUI in Qt, that definitely means using QML.
 It's powerful, flexible, declarative, and allows us to iterate very quickly.
 
-So let's add a `main.qml` next to our two other files in the `src` folder:
+So let's add a `main.qml` file in a `qml` folder:
 ```qml,ignore
 {{#include ../../../examples/qml_minimal/qml/main.qml:book_main_qml}}
 ```
```

Comment:
furhter down there is `used in Qts property binding system` should this be `used in Qt's property binding system`?

Reply:
I never quite got all the rules of the apostrophe in english, it's weirdly similar yet different to German, so I'll trust you on this one ;)

Reply:
I normally get it wrong too though :sweat_smile: maybe this is right ...